MapboxSceneKit/TerrainNode.swift

```diff
@@ -107,10 +107,16 @@ open class TerrainNode: SCNNode {
     /// - Returns: Vector position should be converted from the terrain local space to the world space (or another node's corrdinate space, as needed).
     @objc public func positionForLocation(_ location: CLLocation) -> SCNVector3 {
         let xz = coordinates(location: location)
+        let z = heightForLocalPosition(SCNVector3(xz.x, 0.0, xz.z))
```

Comment:
I don't *love* these variable names even though I get what they convey. if nothing else comes to mind that has more vowels in it, then it's fine.